internal/controlplane/handlers_repositories.go

```diff
@@ -322,21 +322,24 @@ func (s *Server) ListRemoteRepositoriesFromProvider(
 	}
 
 	for _, provider := range provs {
+		// keep linter happy
+		// TODO: this should not be necessary post Go 1.22 - update linter
+		p := provider
```

Comment:
Don't we already use and enforce go 1.22?

Reply:
See the comment I left in this code in my most recent commit - the gosec authors have decided not to get rid of this warning until all versions prior to 1.22 are unsupported.